Multisig/Presentation/Add Safe/SafariViewController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SafariViewController.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 15.04.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+import SwiftUI
+
+struct SafariViewController: UIViewControllerRepresentable {
```

Comment:
Have you tried to use UIViewRepresentable instead?

Reply:
No, I thought because this is a view controller, so I used the controller representable ...